src/Stratis.Bitcoin.Features.PoA/PoABlockHeader.cs

```diff
@@ -0,0 +1,52 @@
+﻿using NBitcoin;
+using NBitcoin.Crypto;
+
+namespace Stratis.Bitcoin.Features.PoA
+{
+    public class PoABlockHeader : BlockHeader
+    {
+        private BlockSignature blockSignature;
+
+        public BlockSignature BlockSignature
+        {
+            get => this.blockSignature;
+            set => this.blockSignature = value;
+        }
+
+        public override void ReadWrite(BitcoinStream stream)
+        {
+            base.ReadWrite(stream);
+            stream.ReadWrite(ref this.blockSignature);
```

Comment:
I guess we could have a comment at the top of this class to explain why we add the block signature here and why we exclude it from the hash calculation.

Reply:
fixed. 

didn't add info why we exclude it from hash calculation- well, because there is no reason to do so.